test/markdownlint-test.js

```diff
@@ -1032,6 +1032,36 @@ tape("configMultiple", (test) => {
     });
 });
 
+tape("configMultipleWithRequireResolve", (test) => {
+  test.plan(2);
+
+  try {
+    fs.symlinkSync(
+      path.resolve(__dirname, "./config/pseudo-package"),
+      path.resolve(__dirname, "../node_modules/test"),
```

Comment:
I think I wasn't clear. Instead of adding to root/node_modules, can we add to root/test/node_modules? That has the benefit of not touching the "real" one and also showing up in git status if something goes wrong.

Reply:
IC, IC, yeah I misunderstood what you were suggesting. I moved the symlink to `root/test/node_modules` now. What we can do as well is just to commit `root/test/node_modules/pseudo-package` instead and thus avid the symlinking complexity. This, however, whould require some change in `.gitignore` so I decided to not do this without your pre-approval.